import.ps1

```diff
@@ -29,7 +29,7 @@ if ($server.Databases[$db_name]) {
 }
 
 foreach ($dir In $("schema", "procedure", "data", "migration")) {
-  $scripts = Get-ChildItem -Path src/$dir -Filter "*.sql"
+  $scripts = Get-ChildItem -Path src/$dir -Filter "*.sql" | Sort-Object { [regex]::Replace($_.Name, '\d+', { $args[0].Value.PadLeft(20) }) }
```

Comment:
Good catch! Whilst this solution works, it may fail in other scenarios and hide potential errors. e.g. the fixed numeric value 20 for padding or if the prefixes will contain leading zeros.
Therefore and since we know the prefix have to start with a number and an underscore by convention, we can simplify this and be more explicit by using `_` as the split delimitir and cast it to an integer before sorting. That also enforces our intention to follow the convention, so that if the explicit cast fails, the user will see a clear error indicating there is something wrong with one of the migration scripts' names.
I'll add a fix for this in a separate PR where I refactor the import and export scripts. Something will this will do:
```powershell
function GetMigrationScripts {
  return Get-ChildItem "src/migration" -Filter "*.sql" | Sort-Object {[int]($_ -split '_')[0]} | ForEach-Object {$_}
}
```

Reply:
No longer relevant after rebasing the pr on master.